.github/workflows/call-speedup-offline-packags.yaml

```diff
@@ -7,28 +7,25 @@ jobs:
       strategy:
         fail-fast: false
         matrix:
-          file_name: [ files-amd64, images-amd64, files-arm64, images-arm64, os-pkgs-centos7, os-pkgs-kylinv10, os-pkgs-redhat8, os-pkgs-redhat7 ]
+          file: [ files-amd64, images-amd64, files-arm64, images-arm64, os-pkgs-centos7, os-pkgs-kylinv10, os-pkgs-redhat8, os-pkgs-redhat7 ]
       permissions:
         packages: write
         contents: read
       steps:
         - uses: actions/checkout@v3
           with:
             fetch-depth: 0
-        - uses: wei/wget@v1
-          with:
-            args: -qO- https://httpbin.org/get
-        - name: wget-file
+        - name: download-file
           run: |
             GIT_TAG=$(git describe --tags --abbrev=8 --dirty)
```

Comment:
If this will only be triggered when the tag is created, then you can actually use the `${{  github.ref_name }}`  or `GITHUB_REF ` parameter.

https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables